### ISSUE TITLE
Why not support no_repeat_ngram_size?

### ISSUE THREAD

**Issue by col-in-coding** [triton-inference-server/tensorrtllm_backend#200]
no_repeat_ngram_size is available in Tensorrt-LLM, so why not support in tensorrtllm_backend?

**Comment by byshiue** [comment]
Because it is not supported in c++ runtime now. Set it as a feature request.

**Comment by ncomly-nvidia** [comment]
This needs to be added to the Triton backend.

**Comment by col-in-coding** [comment]
> This needs to be added to the Triton backend.

Is it possible to allow adding custom logits processors?

**Comment by ZihanLiao** [comment]
Does tensorrtllm_backend support no_repeat_ngram_size now?
